content/influxdb/cloud/reference/release-notes/influx-cli.md

```diff
@@ -0,0 +1,59 @@
+---
+title: influx CLI release notes
+description: Important changes and and what's new in each version of the influx command line interface (CLI).
+weight: 103
+menu:
+  influxdb_cloud_ref:
+    parent: Release notes
+    name: influx CLI 
+---
+
+## v2.1.0 [2021-07-29]
+
+## New repository
+
+This is the initial release of the `influx` CLI from the `influxdata/influx-cli` GitHub repository.
+
+## Breaking changes
+
+### `influx write` skip-header parsing
+
+The `write` command no longer supports `--skipHeader` as short-hand for `--skipHeader 1`.
+This change was made to simplify our CLI parser.
+
+### Stricter input validation for `influx template` commands
+
+The `apply`, `export`, and `stacks` commands now raise errors when CLI options fail to parse instead of silently discarding bad inputs.
+This change was made to help users debug when their commands fail to execute as expected.
+
+### Server-side template summarization and validation
+
+The `template` and `template validate` commands now use an API request to the server to perform their logic, instead of performing the work on the client-side.
+Offline summarization and validation is no longer supported.
+This change was made to avoid significant code duplication between `influxdb` and `influx CLI`, and to allow server-side template logic to evolve without requiring coordinated CLI changes.
+
+### `influx stacks --json` output conventions
+
+The output of `influx stacks --json` previously used an UpperCamelCase naming convention for most keys.
+The command now uses lowerCamelCase consistently for all objects keys, matching the schema returned by the API.
+
+## Features
+
+- Add global `--http-debug` flag to all `influx` commands to help inspect communication with InfluxDB servers.
+- Add [`bucket-schema` commands](/influxdb/cloud/reference/cli/influx/bucket-schema/) to manage explicit measurement schemas in InfluxDB Cloud.
```

Comment:
Should we rearrange this line to put `(Cloud only)` or something similar at the front? To match the `(OSS only)` on other lines, so users can immediately see

Reply:
yes, thank you, @danxmoran